experiment/runner.py

```diff
@@ -172,7 +199,10 @@ def run_fuzzer(max_total_time, log_filename):
         logs.error('Fuzz target binary not found.')
         return
 
-    _unpack_clusterfuzz_seed_corpus(target_binary, input_corpus)
+    if environment.get('CUSTOM_SEED_CORPUS'):
```

Comment:
Maybe the variable should be `CUSTOM_SEED_CORPUS_DIR`?

Reply:
fixed